VehicleUI/src/main/java/com/drivequant/drivekit/vehicle/ui/odometer/fragment/OdometerVehicleDetailFragment.kt

```diff
@@ -113,12 +111,8 @@ class OdometerVehicleDetailFragment : Fragment(), OdometerDrawableListener {
     }
 
     private fun initVehicle(context: Context) {
-        viewModel.getVehicleDrawable(context)?.let { drawable ->
-            Glide.with(context)
-                .load(drawable)
-                .apply(RequestOptions.circleCropTransform())
-                .placeholder(drawable)
-                .into(binding.spinnerItem.imageItem)
+        viewModel.getVehicleDrawableRes().let { drawable ->
+            binding.spinnerItem.imageItem.setImageResource(drawable)
```

Comment:
Je crois qu'on peut simplifier :
```suggestion
        binding.spinnerItem.imageItem.setImageResource(viewModel.getVehicleDrawableRes())
```